[2024-01-31T22:57:22.279+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: challenger_leaderboard.tf_to_df manual__2024-01-31T22:57:19.411377+00:00 [queued]>
[2024-01-31T22:57:22.285+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: challenger_leaderboard.tf_to_df manual__2024-01-31T22:57:19.411377+00:00 [queued]>
[2024-01-31T22:57:22.285+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-01-31T22:57:22.296+0000] {taskinstance.py:2191} INFO - Executing <Task(_PythonDecoratedOperator): tf_to_df> on 2024-01-31 22:57:19.411377+00:00
[2024-01-31T22:57:22.302+0000] {standard_task_runner.py:60} INFO - Started process 80 to run task
[2024-01-31T22:57:22.303+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'challenger_leaderboard', 'tf_to_df', 'manual__2024-01-31T22:57:19.411377+00:00', '--job-id', '212', '--raw', '--subdir', 'DAGS_FOLDER/python_tft_tier.py', '--cfg-path', '/tmp/tmpt20psnox']
[2024-01-31T22:57:22.304+0000] {standard_task_runner.py:88} INFO - Job 212: Subtask tf_to_df
[2024-01-31T22:57:22.375+0000] {task_command.py:423} INFO - Running <TaskInstance: challenger_leaderboard.tf_to_df manual__2024-01-31T22:57:19.411377+00:00 [running]> on host 964619f82b50
[2024-01-31T22:57:22.482+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='challenger_leaderboard' AIRFLOW_CTX_TASK_ID='tf_to_df' AIRFLOW_CTX_EXECUTION_DATE='2024-01-31T22:57:19.411377+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-31T22:57:19.411377+00:00'
[2024-01-31T22:57:22.569+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-01-31T22:57:22.587+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=challenger_leaderboard, task_id=tf_to_df, execution_date=20240131T225719, start_date=20240131T225722, end_date=20240131T225722
[2024-01-31T22:57:22.636+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-31T22:57:22.664+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
